src/backend/distributed/planner/multi_master_planner.c

```diff
@@ -207,10 +207,6 @@ BuildSelectStatementViaStdPlanner(Query *masterQuery, List *masterTargetList,
 	remoteScan->custom_scan_tlist = copyObject(masterTargetList);
 	remoteScan->scan.plan.targetlist = copyObject(masterTargetList);
 
-	/* probably want to do this where we add sublinks to the master plan */
```

Comment:
what is the reason for removing this? To be more specific, I expected to see we set `masterQuery->hasSubLinks` somewhere else, but we don't anymore?

Reply:
We do in BuildJobQuery. Both masterQuery and jobQuery are created by BuildJobQuery.